Documentation/ViewDesigner/CommonSteps.md

```diff
@@ -0,0 +1,131 @@
+# Common Steps
+
+## Enabling Time Range Filter
+View Designer has a built-in default Time Range filter, however, in Workbooks this setting is not enabled by default. Workbooks does allow users to create their own Time Range filters that might be more applicable to their data logs. The steps to generate the filter are listed below:
+
+Select the <strong>Add parameters</strong> option
+![Add Param](./Examples/AddParam.png)
+
+The default Style is set to <strong>Pills</strong>
+
+Select the <strong>Add Parameter</strong> button
+
+![Add Parameter](./Examples/AddParameter.png)
+
+From the sidebar menu, in the <strong>* Parameter name</strong> textbox, type “TimeRange”
+
+Set <strong>Parameter Type</strong> as “Time Range Picker”
+
+Select the “Required?” checkbox
+![Parameter Menu](./Examples/ParamMenu.png)
+
+Save the parameter in the upper left hand corner of the sidebar menu
+
+You can leave the dropdown as “unset” by default or select a default TimeRange value, such as 24 hours.
+
+Select <strong>Done Editing</strong>
+
+Parameters can be utilized in queries by adding curly braces {} around your parameter name.
+
+More details on parameters can be found in the [Workbooks Documentation on Parameters](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Parameters/Parameters.md).
+
+## Updating Queries for with the TimeRange Parameter
+<strong>Option 1: Select TimeRange from the Time Range Dropdown for each query</strong>
+![Time Parameter](./Examples/TimeParam.png)
+
+<strong>Option 2: Update your KQL queries</strong>

```

Comment:
nit: prefer markdown specific syntax like `** around stuff to get **bold** instead of using `<strong>` html tags

Reply:
there seems to be a mix here of sometimes strong, sometimes \**, sometimes \### headings and strong, etc.  mixing/matching markdown and html will make for strange formatting in various places that display markdown